### PR TITLE
Last batch of Gradle file quotemark style changes

### PR DESCRIPTION
**Pull request by binkley**
The first PR only addressed `build.gradle` files.  This PR addresses the remainder, if I didn't miss any.

```
$ find -name \*.gradle | grep "'"
(Exit 1) grep --color=auto "'"
```

My `.bashrc` includes:
```
function __err_trap {
    echo -e "\e[1;31m(Exit $?)\e[0;35m $BASH_COMMAND\e[0m" >&2
}
trap __err_trap ERR
```
Hence, the "Exit 1" above.

### CODE REVIEW

File: app/gradle/versionInfo.gradle [hunk 1]
```diff
@@ -29,7 +29,7 @@ String getBuildDate() {
 
 String versionInfoFileContents() {
     kotlinLicenseHeader +
-    """ |package batect
+        """ |package batect
```
Comment (charleskorn) [inline app/gradle/versionInfo.gradle:32]: Not sure what's happened here, but this formatting looks a little bit off...

File: gradle/spotless.gradle [hunk 2]
```diff
@@ -31,37 +31,37 @@ ext.licenseText = """
...
     }
 
     kotlinGradle {
-        target '*.gradle.kts'
+        target "*.gradle.kts"
 
         ktlint("0.39.0")
 
-        // Check what values are really required with: find . -name '*.gradle.kts' -type f -exec sed -n 17p {} \; | cut -d' ' -f1 | sort | uniq
+        // Check what values are really required with: find . -name "*.gradle.kts" -type f -exec sed -n 17p {} \; | cut -d" " -f1 | sort | uniq
```
Comment (charleskorn) [inline gradle/spotless.gradle:64]: The single quotes here are important (they prevent the shell from performing globbing)
```suggestion
        // Check what values are really required with: find . -name '*.gradle.kts' -type f -exec sed -n 17p {} \; | cut -d' ' -f1 | sort | uniq
```

File: gradle/utilities.gradle [hunk 3]
```diff
@@ -27,7 +27,7 @@ String runProcess(String... args) {
     def output = process.inputStream.getText()
 
     if (exitCode != 0) {
-        throw new RuntimeException("'${args.join(" ")}' failed with exit code $exitCode. Output was: $output")
+        throw new RuntimeException("\"${args.join(" ")}\" failed with exit code $exitCode. Output was: $output")
```
Comment (charleskorn) [inline gradle/utilities.gradle:30]: Is there a reason to change these? The single quotes seem clearer to me.
```suggestion
        throw new RuntimeException("'${args.join(" ")}' failed with exit code $exitCode. Output was: $output")
```

File: gradle/versioning.gradle [hunk 4]
```diff
@@ -28,14 +28,14 @@ tasks.register("validateVersionMatchesTag") {
         }
 
         if (gitTag != project.version.toString()) {
-            throw new Exception("The tag for this commit is '$gitTag', but the project version is set to '$project.version'.")
+            throw new Exception("The tag for this commit is \"$gitTag\", but the project version is set to \"$project.version\".")
```
Comment (charleskorn) [inline gradle/versioning.gradle:31]: Same as above - probably fine to keep these as single quotes

**Review by charleskorn — APPROVED** [review]
LGTM apart from some nitpicks.

I'm not quite sure what's happened, but your PR seems to have picked up some unrelated changes as well (eg. changes to `.editorconfig`, the GitHub Actions workflow and some Dockerfiles) - would you mind removing those to keep the diff clean?

File: gradle/utilities.gradle (hunk 3 above)
Comment (binkley) [reply gradle/utilities.gradle:30]: I went with consistency (meaning, I used a regex to replace quoting).

File: gradle/versioning.gradle (hunk 4 above)
Comment (binkley) [reply gradle/versioning.gradle:31]: Addressed in next series of commits pushed to this PR.

File: gradle/spotless.gradle (hunk 2 above)
Comment (binkley) [reply gradle/spotless.gradle:64]: Addressed in next series of pushes to this PR.

File: app/gradle/versionInfo.gradle (hunk 1 above)
Comment (binkley) [reply app/gradle/versionInfo.gradle:32]: This seemed to resolved itself after I merged from upstream.

**Comment by binkley** [comment]
I'm glad you're nitpicky.  Collaborating on OSS GitHub PRs is stretching muscles I haven't used recently enough.

**Comment by binkley** [comment]
This PR has additional changes: Migrating `settings.gradle` to `settings.gradle.kts`. In future, I'll submit PRs from branches of my fork, rather than from `master` so I don't blend changes.

**Comment by charleskorn** [comment]
This looks great, thanks again @binkley.

**Pull request merged**